PersonApi/V1/Controllers/PersonApiController.cs

```diff
@@ -43,5 +47,16 @@ public async Task<IActionResult> GetPersonByIdAsync(Guid id)
 
             return Ok(person);
         }
+
+        [ProducesResponseType(typeof(PersonResponseObject), StatusCodes.Status200OK)]
```

Comment:
Shouldn't this return a 201 (Created)?

Reply:
200 on Swagger - I can make it 201?

Reply:
Swagger has 201 I would say to update it so its 201